starfish/image/_filter/gaussian_high_pass.py

```diff
@@ -40,9 +40,9 @@ def add_arguments(cls, group_parser: argparse.ArgumentParser) -> None:
             help="indicates that the image stack should be filtered in 3d")
 
     @staticmethod
-    def high_pass(
-            image: Union[xr.DataArray, np.ndarray], sigma: Union[Number, Tuple[Number]]
-    ) -> Union[xr.DataArray, np.ndarray]:
+    def high_pass(image: Union[xr.DataArray, np.ndarray],
```

Comment:
Also, stylistically, I prefer:

```
    def high_pass(
        image: Union[xr.DataArray, np.ndarray],
        sigma: Union[Number, Tuple[Number]],
        rescale: bool = False,
    ) -> Union[xr.DataArray, np.ndarray]:
```

Rationale: lining up the arguments to the opening parenthesis causes a lot of spurious and unnecessary changes when the name of the method changes to something that has a different length.